### ISSUE TITLE
Host request timed out

### ISSUE THREAD

**Issue by dimmuboy** [ateodorescu/home-assistant-addons#7]
When I sent command request it returns correct json response but when I try to send host or to configure IPMI connector it returns timeout error.
> http://192.168.6.9:9595/command?params=-I%20lanplus%20-H%20192.168.6.2%20-U%20Administrator%20-P%2095XGFGBC%20bmc%20info
```
{"success":true,"output":"Device ID                 : 17\nDevice Revision           : 1\nFirmware Revision         : 2.32\nIPMI Version              : 2.0\nManufacturer ID           : 11\nManufacturer Name         : Hewlett-Packard\nProduct ID                : 8192 (0x2000)\nProduct Name              : Unknown (0x2000)\nDevice Available          : yes\nProvides Device SDRs      : yes\nAdditional Device Support :\n    Sensor Device\n    SDR Repository Device\n    SEL Device\n    FRU Inventory Device\n"}
```
> http://192.168.6.9:9595/?host=192.168.6.2&port=623&user=Administrator&password=95XGFGBC
```
2023/12/31 17:32:24 [error] 206#206: *34 upstream timed out (110: Operation timed out) while reading response header from upstream, client: 192.168.6.11, server: _, request: "GET /?host=192.168.6.2&port=623&user=Administrator&password=95XGQTGC HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "192.168.6.9:9595"
```

**Comment by ateodorescu** [comment]
Please get the latest ipmi integration and addon and let me know if they work for you.

**Comment by dimmuboy** [comment]
Thank you, it was fixed in v1.2.5 . I tried it yesterday :)
